src/joint.dia.cell.js

```diff
@@ -608,6 +608,14 @@ joint.dia.Cell = Backbone.Model.extend({
     isLink: function() {
 
         return false;
+    },
+
+    startBatch: function(name, opt) {
+        return this.trigger('batch:start', _.extend({}, opt, { element: this, batchName: name }));
```

Comment:
Implemented.
Even though with a modification

```javascript
    startBatch: function(name, opt) {
        if (this.graph) { this.graph.startBatch(name, opt); }
        return this;
    }
```
The return is important because we would return the Graph and not the Cell instead, breaking chaining operations.